### DISCUSSION TITLE
data problem

### DISCUSSION THREAD

**Discussion by ladraoui** [geeksblabla/blanat#60] [Q&A] [answered]
how can i ensure that a city from dataset missing some product's kind and than there is no logic for comparing the total without ensuring that all cities have a combination of all products in the dataset (because it is a random choice)

**Comment by YassineOsip** [comment, marked as answer]
That's true, but I believe we should simply embrace the challenge and enjoy it.

**Comment by ladraoui** [comment]
ok should i supose that data is complete and work on it ??

**Reply by YassineOsip** [reply]
Yes, Look at Gen.py merged lately by @chermehdi to understand how data getting generated.
